### PR TITLE
{bio}[foss/2018a] IDBA-UD v1.1.3

### PR DESCRIPTION
**Pull request by wpoely86**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/i/IDBA-UD/IDBA-UD-1.1.3-foss-2018a.eb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+source_urls = ['https://github.com/loneknightpy/idba/releases/download/%(version)s']
+sources = ['idba-%(version)s.tar.gz']
+
+prebuildopts = './configure && '
+
+# we delete every .o file which is left in bin folder
+buildopts = ' && rm -fr bin/*.o && rm -fr bin/Makefile*'
+
+files_to_copy = ["bin", "script", "ChangeLog", "NEWS"]
```
Comment (boegel) [inline easybuild/easyconfigs/i/IDBA-UD/IDBA-UD-1.1.3-foss-2018a.eb]: @wpoely86 Why not copy the `README.md` anymore?

File: easybuild/easyconfigs/i/IDBA-UD/IDBA-UD-1.1.3-foss-2018a.eb [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+ to form long contigs in both low-depth and high-depth regions."""
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+
+source_urls = ['https://github.com/loneknightpy/idba/releases/download/%(version)s']
+sources = ['idba-%(version)s.tar.gz']
+
+prebuildopts = './configure && '
+
+# we delete every .o file which is left in bin folder
```
Comment (boegel) [inline easybuild/easyconfigs/i/IDBA-UD/IDBA-UD-1.1.3-foss-2018a.eb]: + Makefile

File: easybuild/easyconfigs/i/IDBA-UD/IDBA-UD-1.1.3-foss-2018a.eb [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+buildopts = ' && rm -fr bin/*.o && rm -fr bin/Makefile*'
```
Comment (boegel) [inline easybuild/easyconfigs/i/IDBA-UD/IDBA-UD-1.1.3-foss-2018a.eb]: `&& rm -rf bin/*.o bin/Makefile*`

File: easybuild/easyconfigs/i/IDBA-UD/IDBA-UD-1.1.3-foss-2018a.eb [hunk 4]
```diff
@@ -0,0 +1,41 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+
+# "make install" doesnt copy all the compiled binaries so we use the "MakeCp" easyblock
+# to be sure everything is copied and we run ./configure in premakeopts
```
Comment (boegel) [inline easybuild/easyconfigs/i/IDBA-UD/IDBA-UD-1.1.3-foss-2018a.eb]: `premakeopts` -> `prebuildopts`

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2104.delcatty.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/22f81baaf38702c3a198dbc8780c5724 for a full test report.

File: easybuild/easyconfigs/i/IDBA-UD/IDBA-UD-1.1.3-foss-2018a.eb (hunk 1 above)
Comment (wpoely86) [reply easybuild/easyconfigs/i/IDBA-UD/IDBA-UD-1.1.3-foss-2018a.eb]: It's not in the tarball anymore...

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
nic1 - Linux centos linux 7.5.1804, AMD Opteron(tm) Processor 6134, Python 2.7.5
See https://gist.github.com/b7be88c76690aa55c98fd34614a4f8d2 for a full test report.

File: easybuild/easyconfigs/i/IDBA-UD/IDBA-UD-1.1.3-foss-2018a.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/i/IDBA-UD/IDBA-UD-1.1.3-foss-2018a.eb]: Yeah, why would it... Fine then.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2689.swalot.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/d91e8a05b32208d46cda6021b8d66b75 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2104.delcatty.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/be3eaec58dee47669dc7c89d6a88dbfc for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3171.skitty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/a1ef1b1397dcee62a691e31cf2f67855 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @wpoely86!

**Pull request merged**